reactDashboard/src/components/SingleValue/SingleValueBar.jsx

```diff
@@ -102,22 +102,22 @@ function SingleValueBar({ currentTotalsHealthBoardDataset = null }) {
       </div>
       <div className="p-2 single-value-container">
         <SingleValue
-          id="dailyFatalities"
-          title="DAILY FATALITIES"
+          id="dailyDeaths"
+          title="DAILY DEATHS"
           subtitle={guardMissingData(
-            getRelativeReportedDate(dailyFatalities.date)
+            getRelativeReportedDate(dailyDeaths.date)
           )}
-          value={guardMissingData(dailyFatalities.value)}
-          tooltip="These are the fatalities reported on the above day, and updated after 2pm daily (can be delayed because of data fetching)."
+          value={guardMissingData(dailyDeaths.value)}
+          tooltip="These are the deaths reported on the above day, and updated after 2pm daily (can be delayed because of data fetching)."
         />
       </div>
       <div className="p-2 single-value-container">
         <SingleValue
-          id="totalFatalities"
-          title="TOTAL FATALITIES"
+          id="totalDeaths"
+          title="TOTAL DEATHS"
           subtitle={SUBTITLE_TOTAL}
-          value={guardMissingData(totalFatalities.value)}
-          tooltip="These are the total number of fatalities where COVID-19 is noted on the Death Certificate since records began on 28 February, 2020."
+          value={guardMissingData(totalDeaths.value)}
+          tooltip="These are the total number of deaths where COVID-19 is noted on the death Certificate since records began on 28 February, 2020."
```

Comment:
lowercase c to match the lowercase d? :)

Reply:
✅